lib/expression/callbacks/standard.ex

```diff
@@ -1434,4 +1434,32 @@ defmodule Expression.Callbacks.Standard do
 
     rem(integer1, integer2)
   end
+
+  @doc """
+  Appends items from one list to another list.
+  """
+  @expression_doc expression: "append([\"A\", \"B\"], [\"C\", \"B\"])",
+                  result: ["A", "B", "C", "B"]
+  def append(ctx, first_list, second_list) do
+    [first_list, second_list] = eval_args!([first_list, second_list], ctx)
+
+    Enum.concat(first_list, second_list)
+  end
+
+  @doc """
+  Deletes an element from a map by the given key.
+  """
+  @expression_doc expression: "delete(contact, \"gender\")",
+                  context: %{
+                    "contact" => %{
```

Comment:
Can we use a different example? Using `contact` here seems to suggest the you can remove fields forma contact profile with this function

Reply:
good point!